locales/en-US/notes.properties

```diff
@@ -23,6 +32,11 @@ disableSync=Disable Sync
 # structure doesn't work for your locale, you can translate this as "Last sync:
 # {date}".
 syncComplete=Synced {date}
+syncCompleteAt=Synced at {date}
```

Comment:
Is `{date}` a date or a time?

Reply:
It is the same as for Saved at (basically a time)

Reply:
```
# LOCALIZATION NOTE (savedComplete): {date} is the time of saving. If this
# structure doesn't work for your locale, you can translate this as "Saved (
# {date})".
savedComplete=Saved at {date}
```

Reply:
Can you give me an example? Is it just the time, or date+time?

Reply:
Sorry for adding this confusion, we already had one string like this and I reused the note. @ryanfeeley is gonna add a mockup of that this should look like below

Reply:
@flodolo Because it should not be possible to see a timestamp from a previous day (as it syncs on open) we can just show the time. Updated the mockup. https://mozilla.invisionapp.com/d/main#/console/11122662/235284916/preview

Reply:
So, if I'm understanding this correctly: in most cases it will be a time, but it still could be a date if I open Firefox after a few days. If that's the case, how does that work for English?

Reply:
I feel like it is not `{date}` anymore, but it is `{time}` ? @flodolo should we change the string?

![image](https://user-images.githubusercontent.com/128755/33276415-83364112-d363-11e7-9fa6-c01ba40f18dd.png)

Reply:
If you're sure that's always only a time (I tried to figure it out the other day on Slack), probably yes.

Reply:
@flodolo I think we are gonna go with `{time}` we cannot fit the full date in there. cc @ryanfeeley

Reply:
@flodolo should i do `syncCompleteAt2` and `syncComplete2` to signal that we changed these ?

Reply:
That, or syncCompleteTime

Reply:
Thanks so much!